src/lang/CMakeLists.txt

```diff
@@ -12,6 +12,12 @@ add_custom_command(
   COMMENT "Generating C++ files out of .po files."
   VERBATIM
   )
+add_custom_command(
+  OUTPUT "${PROJECT_ROOT_DIR}/lib/WUI/resources/fsdata_wui_local.c"
+  COMMAND "${Python3_EXECUTABLE}" "${PROJECT_ROOT_DIR}/utils/wui.py"
+  COMMENT "Generating fsdata_wui_local raw data from WUI resources."
+  VERBATIM
+  )
```

Comment:
Why is it inside `src/lang`? 

/suggestion/ If I am not missing something, please move to a more appropriate location.

Reply:
My mistake. I will be moved into it's own folder. Sorry about that.